src/status_im2/contexts/add_new_contact/views.cljs

```diff
@@ -55,68 +55,66 @@
-         [rn/view (style/container-outer)
-          [rn/view style/container-inner
-           [quo/text (style/text-title)
-            (i18n/label :t/add-a-contact)]
-           [quo/text (style/text-subtitle)
-            (i18n/label :t/find-your-friends)]
-           [quo/text (style/text-description)
-            (i18n/label :t/ens-or-chat-key)]
-           [rn/view style/container-text-input
-            [rn/view (style/text-input-container invalid?)
-             [rn/text-input
-              (merge (style/text-input)
-                     {:default-value  (or scanned @default-value input)
-                      :placeholder    (i18n/label :t/type-some-chat-key)
-                      :on-change-text (fn [v]
-                                        (reset! default-value v)
-                                        (debounce/debounce-and-dispatch
-                                         [:contacts/set-new-identity v nil]
-                                         600))})]
-             (when show-paste-button?
-               [quo/button
-                (merge style/button-paste
-                       {:on-press
-                        (fn []
-                          (reset! default-value @clipboard)
-                          (rf/dispatch
-                           [:contacts/set-new-identity @clipboard nil]))})
-                (i18n/label :t/paste)])]
-            [quo/button
-             (merge style/button-qr
-                    {:on-press #(rf/dispatch
-                                 [::qr-scanner/scan-code
-                                  {:handler :contacts/qr-code-scanned}])})
-             :i/scan]]
-           (when invalid?
-             [rn/view style/container-invalid
-              [quo/icon :i/alert style/icon-invalid]
-              [quo/text style/text-invalid
-               (i18n/label (or msg :t/invalid-ens-or-key))]])
-           (when (= state :valid)
-             [found-contact public-key])]
-          [rn/view
-           [quo/button
-            (merge (style/button-view-profile state)
-                   {:on-press
-                    (fn []
-                      (reset! clipboard nil)
-                      (reset! default-value nil)
-                      (rf/dispatch [:contacts/clear-new-identity])
-                      (rf/dispatch [:navigate-back])
-                      (rf/dispatch [:chat.ui/show-profile public-key ens]))})
-            (i18n/label :t/view-profile)]]]]))))
+
+        [bottom-sheet-screen/view
+         (fn []
+           [rn/view (style/container-outer)
+            [rn/view style/container-inner
+             [quo/button
+              (merge (style/button-close)
+                     {:on-press
+                      (fn []
+                        (reset! clipboard nil)
+                        (reset! default-value nil)
+                        (rf/dispatch [:contacts/clear-new-identity])
+                        (rf/dispatch [:navigate-back]))}) :i/close]
```

Comment:
`close` parameter from `bottom-sheet-screen` should be used to navigate back, because it executes an animation to the background layer

Reply:
Thanks @OmarBasem! Done.

Reply:
@erikseppanen there is a recent commit into develop that simplifies the API for bottom sheet screen. You don't need to wrap the component. You just need to pass `:options   {:sheet? true}` in navigation options.

Reply:
Thanks @OmarBasem! Done